### ISSUE TITLE
Validation/cls Loss and Turning Off Data Augmentation

### ISSUE THREAD

**Issue by TerryYeong** [ultralytics/yolov5#6996]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I have question regarding the loss shown in figure below, increasing losses indicates overfitting. However, my classification loss for 140 epochs shows sign of everlasting decreasing trend, does this indicate underfitting?
![Val loss](https://user-images.githubusercontent.com/72775382/158515184-c2526608-f3bd-47be-b583-9530bf8628be.PNG)
If i wish to turn off the data augmentation, do i just need to turn the augmentation parameter to zero as shown below?
![Data augmentation](https://user-images.githubusercontent.com/72775382/158515459-487a0577-3adb-4765-b527-c3c8183b0322.PNG)

**Comment by glenn-jocher** [comment]
@TerryYeong overfitting occurs when validation loss stops decreasing and starts to increase.

Yes, to turn off augmentation simply set all augmentation hyps to zero.
